docs/usage/appsets-in-any-namespace.md

```diff
@@ -0,0 +1,77 @@
+# ApplicationSets in Any Namespace
+
+**Current feature state**: Beta
+
+!!! note
+    This feature is considered beta feature in upstream Argo CD as of now. Some of the implementation details may change over the course of time until it is promoted to a stable status.
+
+Argo CD supports managing `ApplicationSet` resources in namespaces other than the control plane's namespace. Argo CD administrators can define a certain set of namespaces where `ApplicationSet` resources may be created, updated and reconciled in. 
```

Comment:
@svghadi We need a bit more context/background info/concept infor for downstream along with purposes/benefits stating why this features is important and how it helps the users.